src/Downloader.ts

```diff
@@ -130,7 +136,7 @@ class Downloader {
     this.arrayBufferRequestOptions = {
       // HTTP agent pools with 'keepAlive' to reuse TCP connections, so it's faster
       httpAgent: new http.Agent({ keepAlive: true }),
-      httpsAgent: new https.Agent({ keepAlive: true }),
+      httpsAgent: new https.Agent({ keepAlive: true, rejectUnauthorized: !this.insecure }), // rejectUnauthorized: false disables SSL verification
```

Comment:
You mean TLS (not SSL)

Reply:
yes, it disables the SSL certificate verification step in the TLS handshake. So should I just change it to say ```disables TLS```?

Reply:
Yes